src/Symfony/Component/Messenger/EventListener/StopWorkerOnTimeLimitListener.php

```diff
@@ -30,6 +31,10 @@ public function __construct(int $timeLimitInSeconds, LoggerInterface $logger = n
     {
         $this->timeLimitInSeconds = $timeLimitInSeconds;
         $this->logger = $logger;
+
+        if ($timeLimitInSeconds <= 0) {
+            throw new InvalidArgumentException('Time limit must be greater than zero.');
```

Comment:
```suggestion
            throw new InvalidArgumentException(sprintf('Argument $timeLimitInSeconds must be a positive integer, %d passed.', $timeLimitInSeconds));
```

Reply:
I got this one from https://github.com/symfony/symfony/blob/ff56fe80de77f4019d28cbe105ce1973b09b580a/src/Symfony/Component/Messenger/EventListener/StopWorkerOnMessageLimitListener.php#L35